packages/cli/src/commands/run.js

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+
+import execa from 'execa'
+import terminalLink from 'terminal-link'
+
+import { getPaths } from 'src/lib'
+import c from 'src/lib/colors'
+
+export const command = 'run side'
+export const description = 'Run server for api in production'
+export const builder = (yargs) => {
+  yargs
+    .positional('side', {
+      default: 'api',
+      description: 'Which server to start',
+      type: 'string',
+    })
+    .epilogue(
+      `Also see the ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/reference/command-line-interface#run'
+      )}`
+    )
+}
+
+export const handler = async ({ side = 'api' }) => {
+  // Currently we only support api anyway
+  // Not sure if we will support web in the future
+  if (side === 'api') {
+    if (
+      !fs.existsSync(
+        path.join(getPaths().base, 'node_modules', '.bin', 'api-server')
```

Comment:
// We're going to deprecate `api-server`
```suggestion
        path.join(getPaths().base, 'node_modules', '.bin', 'rw-api-server')
```